lib/hanami/application/settings/loader.rb

```diff
@@ -42,11 +42,11 @@ def initialize(*)
         def call(defined_settings)
           load_dotenv
 
-          settings, errors = load_settings(defined_settings)
+          errors = load_settings(defined_settings)
 
           raise InvalidSettingsError, errors if errors.any?
 
-          settings
+          Settings.config.values
```

Comment:
Continuing from my comment above, having `load_settings` here actually return the configured and populated configurable object would mean we don't have this reference to a constant here appearing out of nowhere, which to me feels like a bit of a design smell.